test/e2e/scenarios/start/benefitType.js

```diff
@@ -25,3 +26,32 @@ Scenario('When I enter a non PIP benefit type, I am taken to the download form p
     I.seeInCurrentUrl(paths.identity.downloadAppointeeForm);
 
 });
+
+const benefit = new DataTable(['BenefitType', 'BenefitName']); //
```

Comment:
There's no need to define a DataTable here, you can easily use a traditional for loop. In addition, we have a list of of benefit type here: https://github.com/hmcts/submit-your-appeal/blob/master/steps/start/benefit-type/types.js

Reply:
I don't have issues with DataTable. Either way is fine by me. However, refactoring the code to reuse the types defined in the class suggested by Paul would be cleaner and nicer.

Reply:
To keep everyone happy how about this:
`const benefitTypes = require('steps/start/benefit-type/types');`
...
`Data(Object.values(benefitTypes)).Scenario('When I enter different non PIP benefit types, I am taken to the download form page', (I, current) => {
...
}
`

Reply:
looks very sexy

Reply:
@sivas23 this is the change that @paulgain suggested. It looks to me more elegant than the for loop you added. But it's not a big issue for me. Anyway, let's wait for @paulgain feedback.